ltp/data.py

```diff
@@ -90,28 +99,57 @@ def arguments(self):
         """
         return self._args
 
+    @property
+    def parallelizable(self):
+        """
+        If True, test can be run in parallel.
+        """
+        return self._parallelizable
+
 
-def read_runtest(suite_name: str, content: str) -> Suite:
+PARALLEL_BLACKLIST = [
+    "needs_root",
+    "needs_device",
+    "mount_device",
+    "mntpoint",
+    "resource_file",
+    "format_device",
+    "forks_child"
```

Comment:
I don't think forks child is a very good indicator. A lot of tests just fork a child to test the process of forking/cloning or unshare, cgroup hierarchy etc. Possibly a better one would be if max_runtime > 5 (assuming you want to avoid running resource intensive tests in parallel).